easybuild/easyconfigs/g/GEOS/GEOS-3.5.0_configure_python3_print.patch

```diff
@@ -0,0 +1,63 @@
+--- configure.orig	2016-12-29 17:33:15.229294313 +0100
```

Comment:
@andreas-h is this a patch you came up with, or did you get is somewhere?

Please include a short description on top, and mention the author (any lines above the first `---` line are comments).